### DISCUSSION TITLE
diary/2022/11/08/%EB%8F%84%EB%8C%80%EC%B2%B4-%EB%AC%B4%EC%97%87%EC%9D%84-%ED%96%88%EB%8A%94%EC%A7%80-%EB%AA%A8%EB%A5%B4%EA%B2%A0%EB%8A%94-%ED%95%98%EB%A3%A8

### DISCUSSION THREAD

**Discussion by giscus** [boyinblue/boyinblue.github.io#14] [General]
# diary/2022/11/08/%EB%8F%84%EB%8C%80%EC%B2%B4-%EB%AC%B4%EC%97%87%EC%9D%84-%ED%96%88%EB%8A%94%EC%A7%80-%EB%AA%A8%EB%A5%B4%EA%B2%A0%EB%8A%94-%ED%95%98%EB%A3%A8

2022년 11월 8일 일기

https://boyinblue.github.io/diary/2022/11/08/%EB%8F%84%EB%8C%80%EC%B2%B4-%EB%AC%B4%EC%97%87%EC%9D%84-%ED%96%88%EB%8A%94%EC%A7%80-%EB%AA%A8%EB%A5%B4%EA%B2%A0%EB%8A%94-%ED%95%98%EB%A3%A8.html

**Comment by boyinblue** [comment]
Giscus 처음으로 연동했습니다.
